sunpy/map/tests/test_mapbase.py

```diff
@@ -480,11 +490,50 @@ def test_rotate_invalid_order(generic_map):
 
 
 @skip_wcsaxes
+def test_as_mpl_axes_aia171(aia171_test_map):
+    import wcsaxes  # import here because of skip
```

Comment:
ahh, `skip_wcsaxes` runs when `wcsaxes` can be imported! Is that right? I read it the other way. sorry.